rollup.config.js

```diff
@@ -64,9 +64,8 @@ function rewirePreactSubpackages() {
                 filePath = args.pop();
 
           const { dir } = parsePath(filePath);
-          const pathToPreact = relativePath(dir, './preact');
-
-          return `${importGroup}${pathToPreact}${afterImport}`;
+          const posixPathToPreact = relativePath(dir, './preact').replace(path.sep, path.posix.sep);
```

Comment:
The `'./'` prefix is now gone. On purpose?

Reply:
Yes, as it turns out that prefix was only needed because of the backslash. ..\.. just wasn't interpreted as a proper path, while ./..\.. was. But we never really needed it in the first place.

Reply:
../.. though works absolutely fine

Reply:
Please be mindful of a replace that does not do what you would expect: 

![image](https://user-images.githubusercontent.com/58601/148837166-c33b5c0f-677a-440f-9661-44debd96de82.png)

I know this is necessary in this place, still, to properly replace all occurences you gotta do `a.split(sep).join(otherSep)`.

Reply:
Ah yes I blame my history with C# here. 

I've made the adjustments and squashed the commits. :)